packages/ago/ago.0.2/descr

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
The first line is blank, so the description won't show up using `opam list` etc.